library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -165,6 +165,17 @@ void PlatformBridgeFilter::onDestroy() {
   platform_filter_.instance_context = nullptr;
 }
 
+envoy_stream_intel PlatformBridgeFilter::streamIntel() {
```

Comment:
the method name could use a verb. A few candidates `getStreamIntel()`, `createStreamIntel()`, `retrieveStreamIntel()`, `fetchStreamIntel()` or something like this. Otherwise it reads as a type constructor. Not a hard blocker

Reply:
I agree that's typical and even preferably in many circumstances. But as it happens, this is the convention for naming getters in Envoy.

Reply:
makes sense then 👌